src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  src: url("../src/fonts/Avenir.ttc") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("../src/fonts/Avenir-Book.ttf") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("../src/fonts/Avenir-Heavy.ttf") format("opentype");
+}
+
+form {
+  margin: 20px 0;
+}
+
+.searchBig input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding: 20px 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff no-repeat 25px 24px url("../src/images/Search.svg");
+  background-size: 19px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.searchBig :hover {
```

Comment:
Don't use `:hover` and `:focus` to the whole form. Use them for input pseudo-classes: `.search__input:hover`, etc.